_inc/lib/class-jetpack-wizard.php

```diff
@@ -22,9 +22,6 @@ public static function can_be_displayed() {
 		 *
```

Comment:
I would suggest marking this as deprecated to make things clearer. We cannot really use `apply_filters_deprecated` here since that would trigger some notices on quite a few sites, but we can still make things a bit clearer in the comment I think. 

```suggestion
		 * @deprecated 9.4.0
		 *
```